src/main/java/stream/flarebot/flarebot/util/votes/VoteGroup.java

```diff
@@ -0,0 +1,119 @@
+package stream.flarebot.flarebot.util.votes;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VoteGroup {
+    private int yesVotes = 0;
+    private int noVotes = 0;
+
+    private List<Long> users = new ArrayList<>();
```

Comment:
Does this need to be a list? Can it also be final?

Reply:
Want me to use a set?

Reply:
Would be better in this case